### PR TITLE
ha: handle invalid operate type on worker

### PR DESCRIPTION
**Pull request by WangXiangUSTC**
### What problem does this PR solve? 

if operate type is not `create` or `update` will use `stop` as default

### What is changed and how it works?

only use `stop` operate when op is `stop`, and handle invalid operate type 

### Check List 

Tests 

 - Integration test

### CODE REVIEW

File: dm/master/server.go [hunk 1]
```diff
@@ -1747,7 +1747,7 @@ func (s *Server) OperateMysqlWorker(ctx context.Context, req *pb.MysqlWorkerRequ
 		if resp, err = w.OperateMysqlWorker(ctx, req, s.cfg.RPCTimeout); err != nil {
 			return makeMysqlWorkerResponse(err)
 		}
-	} else {
+	} else if req.Op == pb.WorkerOp_StopWorker {
```
Comment (csuzhangxc) [inline dm/master/server.go]: How about unifying to use `switch .. case`?

File: dm/master/server.go [hunk 2]
```diff
@@ -1764,7 +1764,13 @@ func (s *Server) OperateMysqlWorker(ctx context.Context, req *pb.MysqlWorkerRequ
...
 			s.coordinator.HandleStoppedWorker(w, cfg)
 		}
+	} else {
+		return &pb.MysqlWorkerResponse{
+			Result: false,
+			Msg:    "invalid operate on worker",
+		}, nil
 	}
+
 	return &pb.MysqlWorkerResponse{
```
Comment (lichunzhu) [inline dm/master/server.go]: Are these lines same to `return resp`?

File: dm/master/server.go (hunk 1 above)
Comment (WangXiangUSTC) [reply dm/master/server.go]: update in 0b80cf4

File: dm/master/server.go (hunk 2 above)
Comment (WangXiangUSTC) [reply dm/master/server.go]: yes, update in 0b80cf4

**Pull request merged**